### PR TITLE
Add open-source Tauri App

### PR DESCRIPTION
**Pull request by stefanodevenuto**
This is the link to the project: [Wirefish](https://github.com/stefanodevenuto/wirefish)

- [X] **I have read the [contributing guidelines](contributing.md).**
- [X] Use the following format: `[Title](link) - Description.`
- [X] Add entries to the bottom of the correct category.
- [X] No unnecessary words like `for Tauri`, `a Tauri plugin` and `Super-Fast`.
- [X] Description does not start with `A` or `An`.
- [X] Description is under 24 words.
- [X] Description has no links or parenthesis.
- [X] Use `backticks` when mentioning package names.
- [X] Double-check spelling and grammar.
- [X] No trailing whitespace is added to the end of any file.
- [X] One suggestion per pull request.
- [X] I understand that my entry will be removed if it violates the guidelines.
- [X] I have [signed my commits](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits).

### Plugins/Integrations

- [ ] Works with **Tauri 1.x and onward**.
- [ ] The project is open source and accepts contributions.
- [ ] The repo is at least 30 days old.
- [ ] Documentation is in English.
- [ ] The project is active and maintained.

### Templates

- [ ] Works with **Tauri 1.x and onward**.
- [ ] The repo is at least 30 days old.
- [ ] Documentation is in English.
- [ ] The template provides enough information about how to get started and what's included.
- [ ] The template is pretty different from the existing templates.

### Apps

- [X] The app is original and not too simple.
- [X] The README is in English.
- [X] The app makes a reasonable effort to be fast, lightweight and secure.
- [ ] If the app closed source, evidence of it being built with Tauri is included.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,6 +103,7 @@ A curated collection of the best stuff from the Tauri ecosystem and community.
 - [Dropcode](https://github.com/egoist/dropcode) - Simple and lightweight code snippet manager.
 - [JournalV](https://github.com/ahmedkapro/journalv) - Journaling app for your days and dreams.
 - [GitBar](https://github.com/mikaelkristiansson/gitbar) - System tray app for GitHub reviews.
+- [Wirefish](https://github.com/stefanodevenuto/wirefish) - Multiplatform packet sniffer and analyzer
```
Comment (JonasKruckenberg) [inline README.md]:
```suggestion
- [Wirefish](https://github.com/stefanodevenuto/wirefish) - Cross-platform packet sniffer and analyzer.
```

**Review by JonasKruckenberg — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (JonasKruckenberg) [reply README.md]: Cross-platform is the word that we used for other apps in the list too, I'd make sense to keep the lingo the same.

Comment (stefanodevenuto) [reply README.md]: Sure! I'll fix it

**Review by JonasKruckenberg — APPROVED** [review]

**Comment by JonasKruckenberg** [comment]
Thanks for being part of the community!

**Pull request merged**